README.md

```diff
@@ -235,6 +235,13 @@ Sidekiq.configure_server do |config|
 end
 ```
 
+For applications running sidekiq < `6.5.0` but want to add the snippet, you can guard the snippet with something like this and remove it once you upgrade sidekiq:
```

Comment:
Why wound anyone want to add the snippet before upgrading to 6.5.0+?

Reply:
Because they might forget to add this snippet after upgrading to Sidekiq `6.5.0`, since it's related to Prosopite not Sidekiq. They can add it and comment it out and have a TODO comment or something. But same idea.